src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -199,3 +199,24 @@
     border: none;
   }
 }
+
+.suggest-widget {
+  position: relative;
+
+  &.visible::after {
```

Comment:
I'm appending the box onto the `suggest-widget` class, which is what's used by the monaco editor package. Also additionally using the `visible` class to only display it when the suggestion window is visible

Reply:
instead of this, could we create an element on the code_editor?

for example

```
_editorDidMount = (editor: monaco.editor.IStandaloneCodeEditor, __monaco: unknown) => {
 ...
  const suggestWidget = editor.getContribution('editor.contrib.suggestController')?.widget;
  if (suggestWidget) {
    const originalRender = suggestWidget.render;
    suggestWidget.render = (element: HTMLElement) => {
      originalRender.call(suggestWidget, element);
      
      let messageElement = element.querySelector('.suggest-widget-message');
      if (!messageElement) {
        messageElement = document.createElement('div');
        messageElement.className = 'suggest-widget-message';
        messageElement.textContent = 'Tab to insert, ESC to close window';
        element.appendChild(messageElement);
      }
    };
  }

...
};

Reply:
I tried this method out first but it didn't work for me. I could try to revisit it, but part of the issue could be that the version of monaco that we're using is old so it may not work the same compared to if it was up to date.

Reply:
i see.

for  the styling though could we use the standard values?